src/Previewer/previewer.vala

```diff
@@ -118,7 +119,7 @@ namespace Workbench {
         var end = section.get_end_location();
         this.css_parser_error(error.message, (int)start.lines, (int)start.line_chars, (int)end.lines, (int)end.line_chars);
       });
-      this.css.load_from_data (content, -1);
+      this.css.load_from_data (content.data);
```

Comment:
Not sure where this change comes from.
If I revert it though, I get the following error:

```
../../../../../../../../../Projekte/github.com/sonnyp/Workbench/src/Previewer/previewer.vala:122.32-122.38: error: Argument 1: Cannot convert from `unowned string' to `unowned uint8[]'
  122 |       this.css.load_from_data (content, -1);
      |                                ^~~~~~~      
../../../../../../../../../Projekte/github.com/sonnyp/Workbench/src/Previewer/previewer.vala:122.7-122.43: error: 1 extra arguments for `void Gtk.CssProvider.load_from_data (uint8[])'
  122 |       this.css.load_from_data (content, -1);
```